README.md

```diff
@@ -48,190 +27,413 @@ The cats table is populated with the following data:
+  * Similar to `>` in Python
+* `>=` ("greater than or equal to")
+  * Similar to `>=` in Python
+* `<` ("less than")
+  * Similar to `<` in Python
+* `<=` ("less than or equal to")
+  * Similar to `<=` in Python
+* `AND`
+  * Similar to `and` in Python
+* `OR`
+  * Similar to `or` in Python
+* `BETWEEN`
+  * Similar to placing a value between two values with `<=` and `and` in Python, e.g. `(2 <= x) and (x <= 5)`
+* `IN`
+  * Similar to `in` in Python
+* `LIKE`
+  * Uses wildcards to find similar strings. No direct equivalent in Python, but similar to some Bash terminal commands.
+
+### `WHERE` Code-Along
+
+In this exercise, you'll walk through executing a handful of common and handy SQL queries. We'll start by giving you an example of what this type of query looks like, then have you type a query specifically related to the `cats` table.
+
+#### `WHERE` with `>=`
+
+For the `=`, `!=`, `<`, `<=`, `>`, and `>=` operators, the query looks like:
+
+```sql
+SELECT column(s) FROM table_name WHERE column_name operator value;
 ```
 
+> Note: The example above is not valid SQL, it is a template for how the queries are constructed
 
-```python
-# select all cats order by age here
+Type this SQL query between the quotes below to select all cats who are at least 5 years old:
+
+```sql
+SELECT * FROM cats WHERE age >= 5;
 ```
 
-This should return the following:
 
 ```python
-[(6, None, None, 'Tabby', None),
- (2, 'Hana', 1, 'Tabby', 1),
- (5, 'Patches', 2, 'Calico', None),
- (1, 'Maru', 3, 'Scottish Fold', 1),
- (3, "Lil' Bub", 5, 'American Shorthair', None),
- (4, 'Moe', 10, 'Tabby', None)]            
+pd.read_sql("""
+
+""", conn)
 ```
 
-When using `ORDER BY`, the default is to order in ascending order. If you want to specify though, you can use `ASC` for "ascending" or `DESC` for "descending." **Let's try to select all of our cats and sort them by age in descending order.**
+This should return:
 
-```python
-cur.execute('''SELECT * FROM cats ORDER BY age DESC;''').fetchall()
-```
+|id  |name     |age    |breed              |owner_id   |
+|----|---------|-------|-------------------|-----------|
+|3   |Lil' Bub |5.0    |American Shorthair |None       |
+|4   |Moe      |10.0   |Tabby              |None       |
 
+#### `WHERE` with `BETWEEN`
 
-```python
-# select cats order by age descending here
-```
+If you wanted to select all rows with values in a range, you _could_ do this by combining the `<=` and `AND` operators. However, since this is such a common task in SQL, there is a shorter and more efficient command specifically for this purpose, called `BETWEEN`.
 
-This should return
+A typical query with `BETWEEN` looks like:
 
-```python
-[(4, 'Moe', 10, 'Tabby', None),
- (3, "Lil' Bub", 5, 'American Shorthair', None),
- (1, 'Maru', 3, 'Scottish Fold', 1),
- (5, 'Patches', 2, 'Calico', None),
- (2, 'Hana', 1, 'Tabby', 1),
- (6, None, None, 'Tabby', None)]      
+```sql
+SELECT column_name(s) FROM table_name WHERE column_name BETWEEN value1 AND value2;
```

Comment:
I wonder if for longer queries if it would be helpful to have new lines to further accentuate each command?

Reply:
Something like https://www.sqlstyle.guide/?

Reply:
That style guide looks great!

For these, I tried expanding them onto multiple lines but ended up preferring the look of all of them being one-line for the sake of consistency across all of the examples. That's the difficulty with educational examples vs. following a style guide designed for realistic queries

PR [here](https://github.com/hoffm386/dsc-filtering-and-ordering/pull/1) against this branch to see what the expanded version would look like